src/app/view-modes/collation/collation.component.ts

```diff
@@ -25,6 +26,24 @@ export class CollationComponent implements OnInit, OnDestroy {
     map(({ page }) => page.id),
   );
 
+  public get addWitnessBtnLabel(): { label: string, additionalClasses: string, title: string, icon: EvtIconInfo } {
+    if (this.witnesses.length > 0) {
+      return {
+        label: '',
```

Comment:
I do not understand the usage of label and title with respect to witnesses length

Reply:
It's just a graphic matter. If no witnesses have been selected for comparison, we display the button with the label, otherwise only the button with the "plus".